examples/sprites/main.cpp

```diff
@@ -96,68 +74,35 @@ int main()
     robot_sprite.add_animation("attack_left", attack_left_animation);
 
     robot_sprite.play(current_animation_state);
+}
 
-    rinvid::RinvidGfx::init();
-
-    while (static_running)
-    {
-        auto start = std::chrono::high_resolution_clock::now();
-
-        handle_events(window, event);
-
-        rinvid::RinvidGfx::clear_screen(0.2F, 0.4F, 0.4F, 1.0F);
-
-        horizontal_delta = 0.0F;
-
-        robot_sprite.draw(delta_time.count());
-
-        handle_inputs(horizontal_delta);
-
-        bool state_changed = handle_animation_state(action, direction, horizontal_delta,
-                                                    robot_sprite.is_animation_finished());
+void SpritesScreen::update(double delta_time)
+{
+    rinvid::RinvidGfx::clear_screen(0.2F, 0.4F, 0.4F, 1.0F);
 
-        if (state_changed == true)
-        {
-            current_animation_state = action + "_" + direction;
-            robot_sprite.play(current_animation_state);
-        }
+    float horizontal_delta = 0.0F;
 
-        robot_sprite.move(rinvid::Vector2<float>{horizontal_delta, 0});
+    robot_sprite.draw(delta_time);
 
-        window.display();
+    handle_inputs(horizontal_delta);
 
-#ifdef __unix__
-        usleep(10000);
-#else
-        Sleep(10);
-#endif
+    bool state_changed = handle_animation_state(action, direction, horizontal_delta,
+                                                robot_sprite.is_animation_finished());
 
-        auto end   = std::chrono::high_resolution_clock::now();
-        delta_time = end - start;
+    if (state_changed == true)
+    {
+        current_animation_state = action + "_" + direction;
+        robot_sprite.play(current_animation_state);
     }
 
-    return 0;
+    robot_sprite.move(rinvid::Vector2<float>{horizontal_delta, 0});
```

Comment:
Maybe we should use pixels per second unit for movement? 

Same goes for other example

Reply:
I would be ok with using pixels per second, but I don't think we should do it as part of this change. This change is about removing references to SFML from examples and using Rinvid's Aplication and Screen classes. That being said, if u want you can make another PR to make the change yourself if you wish.

Reply:
Yeah, that sounds like better idea.